src/app/sitting-records/manage-sitting-records/manage-sitting-records.component.ts

```diff
@@ -46,7 +46,7 @@ export class ManageSittingRecordsComponent implements OnInit {
     private cookies: CookieService,
     protected activatedRoute: ActivatedRoute,
     private formBuilder: FormBuilder,
-    private srWorkFlow: SittingRecordWorkflowService,
+    private srWorkFlow: RecorderWorkflowService,
```

Comment:
rename from srWorkFlow to match the new use of recorderWorkflow

Reply:
Done